lxd/device/nic_bridged.go

```diff
@@ -964,17 +980,6 @@ func (d *nicBridged) setFilters() (err error) {
 	IPv4 := net.ParseIP(d.config["ipv4.address"])
 	IPv6 := net.ParseIP(d.config["ipv6.address"])
 
-	// If parent bridge is unmanaged check that a manually specified IP is available if IP filtering enabled.
```

Comment:
I think this bit should stay in right? Because we will still require `ipv{n}.address` to be set to "none" (and not empty string) at least, when using IP filtering on an unmanaged bridge.